### PR TITLE
Bh WH guidance and change note

### PR DESCRIPTION
**Pull request by benhazell**
Updated guidance for Whitehall users and change note for recent updates.
Need content review before deploying to production.

### CODE REVIEW

File: app/views/publisher_information/how_to_use_publisher.govspeak.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
-Read more about [what Content Publisher can and cannot do](/beta-capabilities).
+##Whitehall users
+
+Content Publisher works differently to Whitehall. If you are familiar with Whitehall there are some differences to be aware of:
+
+- Markdown can be added using a menu to format text
+- Text pasted into the content body will be automatically converted to markdown
+- There is an Insert menu to use to add pictures, contacts, or video embeds
+- Pictures can be any size larger than 960px wide and 640px high 
+- Content has one primary organisation tag. They will be responsible for it
```
Comment (kevindew) [inline app/views/publisher_information/how_to_use_publisher.govspeak.md]: I'm a bit confused by the ". They will be responsible for it" part here

File: app/views/publisher_information/publisher_updates.govspeak.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
+
+### Email links, content data, and guidance updates <span class="govuk-caption-m">16 May 2019</span>
+Added a link to [Content Data](https://content-data.publishing.service.gov.uk/content) from the edit screen so you can easily check the performance of published content.
```
Comment (kevindew) [inline app/views/publisher_information/publisher_updates.govspeak.md]: We should time this with some work tomorrow morning to remove the organisation restriction of this. Perhaps @1pretz1 can pair with Jonathon on it.

File: app/views/publisher_information/publisher_updates.govspeak.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+
+You can now make email links with the link button. Select an email address and the link button will add email markdown.
+
+Updated styling of markdown preview to help you see the difference between editing and preview more easily.
+
+Added [guidance to help users familiar with Whitehall publisher get started](/how-to-use-publisher).
+
+We have also updated the guidance on
```
Comment (kevindew) [inline app/views/publisher_information/publisher_updates.govspeak.md]: You need a line break before these

Screen Shot 2019-05-15 at 16 31 20

File: app/views/publisher_information/publisher_updates.govspeak.md (hunk 2 above)
Comment (1pretz1) [reply app/views/publisher_information/publisher_updates.govspeak.md]: Yup this sounds good, I'll pair with Jonathon when he's free and update here when a pr is ready.

File: app/views/publisher_information/publisher_updates.govspeak.md (hunk 3 above)
Comment (benhazell) [reply app/views/publisher_information/publisher_updates.govspeak.md]: No YOU need a line break

**Comment by 1pretz1** [comment]
A [pr](https://github.com/alphagov/content-publisher/pull/1042) has been approved concerning displaying the content-data link for all orgs, so whenever the content is ready we'll deploy all the things!

**Comment by 1pretz1** [comment]
Closed this pr as the commits have been merged into -> https://github.com/alphagov/content-publisher/pull/1042.

**Pull request closed without merging**